### ISSUE TITLE
is not a valid process=count option

### ISSUE THREAD

**Issue by virajsoni06** [superfly/flyctl#511]
**Describe the bug**
Have successfully deployed a node app on fly. When trying to scale count, getting the below error

```
$ flyctl scale count 6

Error 6 is not a valid process=count option
```

Not sure what's causing this. Any help would be appreciated.

Here's the fly.toml config

```
app = "REDACTED"

kill_signal = "SIGINT"
kill_timeout = 5
processes = []

[build]
  builder = "heroku/buildpacks:20"

[env]
  PORT = "8080"

[experimental]
  allowed_public_ports = []
  auto_rollback = true

[[services]]
  http_checks = []
  internal_port = 8080
  processes = ["app"]
  protocol = "tcp"
  script_checks = []

  [services.concurrency]
    hard_limit = 25
    soft_limit = 20
    type = "connections"

  [[services.ports]]
    handlers = ["http"]
    port = 80

  [[services.ports]]
    handlers = ["tls", "http"]
    port = 443

  [[services.tcp_checks]]
    grace_period = "1s"
    interval = "15s"
    restart_limit = 6
    timeout = "2s"
```

**Comment by StarpTech** [comment]
Hi, I run into the same issue.

**Comment by StarpTech** [comment]
Looks like a bug 

https://github.com/superfly/flyctl/blob/62df371ef350c9034f75216df98795cf66e10d9e/cmd/scale.go#L95

should not be executed. The condition should be `> 1

**Comment by StarpTech** [comment]
The workaround is `fly scale count app=2`

**Comment by amalkh5** [comment]
@simonw @virajsoni06 it has been fixed by this commit 62df371ef350c9034f75216df98795cf66e10d9e

You can install the pre-release by running 
`
$ curl -L https://fly.io/install.sh | sh -s pre
`
